examples/stm32/xively_demo_stm32f4xx_nucleo_eth/Src/xively_client.c

```diff
@@ -756,12 +758,12 @@ connect_cb( const xi_context_handle_t ctx, void* data, xi_state_t state )
         case XI_CONNECTION_STATE_OPEN_FAILED:
         {
             printf( "[%d] Xively: Connection failed to %s:%d, error %d\n",
-                    xi_bsp_time_getcurrenttime_seconds(), conn_data->host,
+                    ( int )xi_bsp_time_getcurrenttime_seconds(), conn_data->host,

```

Comment:
I believe xi_time_t is a long, in which case you can format this in the printf as a %ld and drop the casting (I think)